src/lib/network_pool/indexed_pool.ml

```diff
@@ -29,15 +29,16 @@ type t =
       User_command.With_valid_signature.Set.t Currency.Fee.Map.t
         (** Transactions valid against the current ledger, indexed by fee. *)
   ; all_by_sender:
-      (User_command.With_valid_signature.t F_sequence.t * Currency.Amount.t)
+      ( User_command.With_valid_signature.t F_sequence.t sexp_opaque
```

Comment:
The only time we ever sexp this type is for debugging the data structure. This shouldn't be opaque.